packages/hoc-withtooltip/src/withTooltip.js

```diff
@@ -0,0 +1,102 @@
+import * as React from "react";
+import Tooltip, { alignments } from "@crave/farmblocks-tooltip";
+import wrapDisplayName from "recompose/wrapDisplayName";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import values from "object.values";
+
+const EnhancedDiv = styled.div`
+  position: relative;
+  display: inline-block;
```

Comment:
The `enhanced element with display block` story from `hoc-disabled-tooltip` is no longer working as expected.
Before:
![2018-10-22 10 09 40](https://user-images.githubusercontent.com/1459283/47294195-b9b87980-d5e2-11e8-9a9e-c6bc9a05337c.gif)
After:
![2018-10-22 10 09 56](https://user-images.githubusercontent.com/1459283/47294204-c1781e00-d5e2-11e8-8284-5936ca12ef04.gif)

Take a look at the `Wrapper` styled component from the hoc-disabled. You can move the `displayBlock` prop to here and remove the whole wrapper from there.